extensions/wikia/AdEngine/js/AdEngine2.js

```diff
@@ -9,17 +9,15 @@ define('ext.wikia.adEngine.adEngine', [
 	'wikia.document',
 	'wikia.lazyqueue',
 	'wikia.log'
-], function (
-	adDecoratorLegacyParamFormat,
-	eventDispatcher,
-	adSlot,
-	slotTracker,
-	slotTweaker,
-	registerHooks,
-	doc,
-	lazyQueue,
-	log
-) {
+], function (adDecoratorLegacyParamFormat,
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Function has 9 parameters which is greater than 7 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AExcessiveParameterList)